src/flows/pipes/Visualization/threshold.ts

```diff
@@ -18,31 +18,28 @@ export interface ErrorThreshold extends Threshold {
   fieldType: string
 }
 
+export const lambdaPrefix = '(r) =>'
+
 export const COMMON_THRESHOLD_TYPES = {
   greater: {
     name: 'greater than',
     format: ThresholdFormat.Value,
-    condition: data => `(r) => (r["${data.field}"] > ${data.value})`,
+    condition: data => `${lambdaPrefix} (r["${data.field}"] > ${data.value})`,
```

Comment:
i'm not following. this work feels unrelated, why was this change  made in this PR?

Reply:
See how the `lambdaPrefix` is then used in the:
```
THRESHOLD_TYPES[threshold.type].condition(threshold))
      .map((cond, i) => (i > 0 ? cond.split(lambdaPrefix)[1] : cond))
      .join(' and ')
```

The purpose is to fix this bug: https://github.com/influxdata/ui/issues/4021

Reply:
k so `lambdaPrefix` is just a const (should be `LAMBDA_PREFIX` in that case, and should be applied more uniformly across the codebase in case we change it). But if the intent is to just apply it on locations that might later have to be passed through this split to match this conditional rendering, then we're creating an ill defined coupling between the definition of the condition and code that conditionally mutates it. while you've updated the current dealings, there's nothing enforcing that relationship down the line, meaning these have a good chance of drifting.

since it doesn't seem like you actually want / need the `(r) =>` to be configurable (correct me if i read this wrong), a better interface might be to modify the condition functions to handle the rendering themselves by updating the function signature to something like `condition: (data: Record<"field"|"value", string>, prefix?: string) => string`, or to remove the prefix from the condition interface altogether and just conditionally apply the prefix to the whole string, so... 
```
const thresholds  = '(r) => ` + (THRESHOLD_TYPES[threshold.type].condition(threshold))
      .map((cond, i) => (i > 0 ? cond.split(lambdaPrefix)[1] : cond))
      .join(' and '))
```

Reply:
removed per request.